src/Mono.Android/Android.Runtime/TypeManager.cs

```diff
@@ -13,9 +13,9 @@ public static string LookupTypeMapping (string[] mappings, string javaType)
 			return Java.Interop.TypeManager.LookupTypeMapping (mappings, javaType);
 		}
 
-		public static void RegisterType (string java_class, Type t)
+		public static void RegisterType (string java_class, Type t, bool ignoreExistingRegistration = false)
```

Comment:
Why update an `[Obsolete]` class at all?

Additionally, this class is `public`, so this change is an ABI break. That's Bad™.